spec/config/routes.pending.yml

```diff
@@ -0,0 +1,2463 @@
+---
```

Comment:
Should this be alphabetized (both the top level as well as the second level)?  I'm thinking as it updates in the future, then the diffs will be more predictable.